Mathlib/FieldTheory/PrimitiveElement.lean

```diff
@@ -97,8 +101,8 @@ variable (F)
 
 variable [Algebra F E]
 
--- This is the heart of the proof of the primitive element theorem. It shows that if `F` is
--- infinite and `α` and `β` are separable over `F` then `F⟮α, β⟯` is generated by a single element.
+/-- This is the heart of the proof of the primitive element theorem. It shows that if `F` is
+infinite and `α` and `β` are separable over `F` then `F⟮α, β⟯` is generated by a single element. -/
 theorem primitive_element_inf_aux [IsSeparable F E] : ∃ γ : E, F⟮α, β⟯ = F⟮γ⟯ := by
```

Comment:
The next thing to do is to prove the `IsSeparable` version of the theorem `exists_primitive_element` using `exists_primitive_element_of_finite_intermediateField`, and then this long proof can be removed.

The key ingredient would be [IsGalois.intermediateFieldEquivSubgroup](https://leanprover-community.github.io/mathlib4_docs/Mathlib/FieldTheory/Galois.html#IsGalois.intermediateFieldEquivSubgroup), but that need to be applied to the SplittingField, so some maneuvers would be required.

Reply:
Oh I guess that's circular ... would require refactoring the proof of `card_aut_eq_finrank`, which is definitely out of scope of this PR.

Reply:
Maybe it's related to separable degree of field extension <https://leanprover.zulipchat.com/#narrow/stream/217875-Is-there-code-for-X.3F/topic/Separability.20degree.3F>?

Reply:
Indeed since there is [Normal.algHomEquivAut](https://leanprover-community.github.io/mathlib4_docs/Mathlib/FieldTheory/Normal.html#Normal.algHomEquivAut), so we're done once you show the separable degree of a separable extension is equal to its degree.